test/models/lookup_value_test.rb

```diff
@@ -67,6 +67,20 @@ def valid_attrs2
     end
   end
 
+  test "can create lookup value if match fqdn= does match existing host" do
+    as_admin do
+      domain = FactoryBot.create(:domain)
+      host = FactoryBot.create(:host)
+      fqdn = [host.name, domain.name].join('.')
+      host.primary_interface.name = fqdn
```

Comment:
Can't we set the domain to the interface instead and than it would pick up the right fqdn by itself?
This feels bit like forcing the value to the interface and we wouldn't need to really create the records, we could just stub them as I feel it tests more "primary_interface.name" is going to be called not the part that the fqdn is actually set on the primary interface.

Reply:
Thanks for reviewing this. I have updated the test. It should be a lot better now.